frontend/public/components/utils/kebab.tsx

```diff
@@ -398,16 +399,20 @@ kebabFactory.common = [
   kebabFactory.Delete,
 ];
 
+let kebabActionExtensions: KebabActions[] = [];
+
+subscribeToExtensions<KebabActions>((extensions) => {
```

Comment:
FYI - `KebabActions` is the extension type name, and `KebabAction` is the function for creating `KebabOption` objects.

The naming is confusing, we should use a better name when migrating `KebabActions` extension to a new type.